src/pybind/mgr/influx/module.py

```diff
@@ -77,29 +77,19 @@ def get_df_stats(self):
         df = self.get("df")
         data = []
 
-        df_types = [
-            'bytes_used',
-            'dirty',
-            'rd_bytes',
-            'raw_bytes_used',
-            'wr_bytes',
-            'objects',
-            'max_avail'
-        ]
-
-        for df_type in df_types:
-            for pool in df['pools']:
+        for pool in df['pools']:
+            for key in pool['stats']:
                 point = {
                     "measurement": "ceph_pool_stats",
                     "tags": {
                         "pool_name": pool['name'],
                         "pool_id": pool['id'],
-                        "type_instance": df_type,
+                        "type_instance": key,
                         "fsid": self.get_fsid()
                     },
                     "time": datetime.utcnow().isoformat() + 'Z',
                     "fields": {
-                        "value": pool['stats'][df_type],
+                        "value": pool['stats'][key],
```

Comment:
If we're using pool['stats'][key] here, then let's use a `for key, value in pool['stats'].items()` in the loop and avoid the second lookup